### DISCUSSION TITLE
Config.defaults.ini file

### DISCUSSION THREAD

**Discussion by squowse** [Louisvdw/dbus-serialbattery#1057] [Q&A] [answered]
Does the driver read this file for settings? Or
Is it purely for reference for the user? 
eg if the default file says setting = true and the config.ini file says setting = false.

**Comment by rolandlueders** [comment, marked as answer]
the settings from the config.ini overwrites the values loaded from the config.default.ini. In your example "false" is used

**Reply by squowse** [reply]
Thanks
